### PR TITLE
Add elevator booking and upgrade packages

### PR DESCRIPTION
**Pull request by djensenius**

### CODE REVIEW

**Review by MaggieShan — APPROVED** [review]
lgtm

File: src/services/admin-api.ts [hunk 1]
```diff
@@ -61,3 +49,37 @@ export interface ParkingRegistration {
   color: string;
   contact: string;
 }
+
+export interface ElevatorBooking {
+  id: number;
+  startTime: Date;
+  endTime: Date;
+  unit: number;
+  ownerType: string;
```
Comment (MaggieShan) [inline src/services/admin-api.ts:58]: Now that user type is added, is this still needed? Should it be replaced by user type?

**Review by MaggieShan — COMMENTED** [review]
Is there a difference between owner and user type?

File: src/services/admin-api.ts (hunk 1 above)
Comment (djensenius) [reply src/services/admin-api.ts:58]: Good question! 

One of the requests was that owners can book for tenants in the case of moving in and out. Hence having a type here.

Comment (MaggieShan) [reply src/services/admin-api.ts:58]: Ahh I see, thanks!

**Review by MaggieShan — APPROVED** [review]
lgtm

**Pull request merged**